lib/data/receiver.js

```diff
@@ -37,6 +37,10 @@ class DataTrackReceiver extends DataTrackTransceiver {
     dataChannel.addEventListener('message', event => {
       this.emit('message', event.data);
     });
+
+    dataChannel.addEventListener('close', () => {
```

Comment:
@makarandp0 Can you add:
```js
/**
 * @emits DataTrackReceiver#close
 */
```
to the class documentation? Also, at the end of the file, can you define the "close" event:
```js
/**
 * @event DataTrackReceiver#close
 */
```

Reply:
@makarandp0 Can you add a CHANGELOG.md entry?